soc/nordic/soc.yml

```
@@ -0,0 +1,9 @@
+family:
+  - name: nordic_nrf
+    series:
+      - name: nrf71
+        socs:
+          - name: nrf7120_enga
```

```suggestion
          - name: nrf7120
```
The board can select the revision'd Kconfig but there is only one SoC